css/properties/overflow-wrap.json

```diff
@@ -100,6 +100,108 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "break-word": {
+          "__compat": {
+            "description": "<code>break-word</code>",
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": "70"
```

Comment:
Set to true. 

If you ever see a difference this drastic in implementation numbers, it's probably wrong. If I told you 70 was correct, I apologize.